rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -93,7 +101,8 @@ Since it takes some time for the transactions to be synced to all network peers
 
 ![DAG structure](../.gitbook/assets/rfc004-branching.svg)
 
-Branches may remain unmerged. As mentioned at the beginning of this paragraph, a transaction MUST refer to a previous transaction. It doesn't matter on which branch the latest transaction is published. 
+Branches may remain unmerged. As mentioned at the beginning of this paragraph, a transaction MUST refer to a previous transaction. It doesn't matter on which branch the latest transaction is published.
+**LC** values will overlap between branches.
```

Comment:
I might be wrong, but it looks like we are only referencing 1 head. If that case the part about merging in section 3.3 is no longer relevant?

Reply:
It doesn't hurt either. There may be some content in there that might be relevant for garbage collection.